pages/about/using-wai-material.md

```diff
@@ -92,6 +92,31 @@ Unless otherwise indicated, all WAI material not covered by another license is p
 
 For material under the Creative Commons License, the footer includes the license, such as: "This work is published and licensed under a [Creative Commons Attribution 4.0 International License](https://creativecommons.org/licenses/by/4.0/)."
 
+## Videos
+You may use videos in your presentations, training, etc, if you use the entire video including the ending with the link to w3.org/WAI
```

Comment:
This could be read as either:

* If I use the entire video, I must include the link
* I can only use the entire video and it must include the link

The language is just a wee bit difficult to parse.

Reply:
"if" may be a "provided that".
The question remains to be addressed regarding using portions of videos (and if it's not allowed, to state it.)

Reply:
We want them to use the entire video, though not sure we want to explicitly say they can't use part. Perhaps we'd want to say they have to use the entire video, unless they get permission from us to use an excerpt with appropriate credit. However, right now I'm thinking that's not worth the extra wording here or effort for us to review!

For now, I've got:
> You may use W3C WAI videos in your presentations, training, etc. Please use the entire video, including the ending with the link to w3.org/WAI

We can revise that later if/when we feel it's important to refine it.